### PR TITLE
Coopr 537 Add documantation to job pause/resume

### PR DESCRIPTION
**Pull request by ytolochkevych**

### CODE REVIEW

File: coopr-docs/docs/source/rest/clusters.rst [hunk 1]
```diff
@@ -1231,3 +1231,80 @@ In order to delete the cluster, the client must re-supply those fields.
...
 
 
+Pause a Cluster Job
+=============================
+To pause a cluster job that is currently running, make a POST HTTP request to URI:
+::
+
+ /clusters/{cluster-id}/pause
+
+All the tasks that were running before work was stopped, continue running until the end of execution.
```
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst]: the wording here is a little confusing.  Something like:

Pausing a job prevents additional tasks from being started, but any tasks that are in progress when a job is paused will continue to run until they are completed.

File: coopr-docs/docs/source/rest/clusters.rst [hunk 2]
```diff
@@ -1231,3 +1231,80 @@ In order to delete the cluster, the client must re-supply those fields.
...
+
+Example
+^^^^^^^
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:<user-id>'
+        -H 'Coopr-TenantID:<tenantid>'
+        -H 'Coopr-ApiKey:<apikey>'
+        -X POST
+        -d '{
```
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst]: no post body should be here

File: coopr-docs/docs/source/rest/clusters.rst [hunk 3]
```diff
@@ -1231,3 +1231,80 @@ In order to delete the cluster, the client must re-supply those fields.
...
```
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst]: no post body should be here

File: coopr-docs/docs/source/rest/clusters.rst (hunk 1 above)
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst]: should also include something about who is allowed to pause, and what happens if you pause when there is no running job:

Admins and cluster owners are allowed to pause a cluster job.  If there is no job in progress, the request is a no-op and a 200 is returned.

File: coopr-docs/docs/source/rest/clusters.rst [hunk 4]
```diff
@@ -1231,3 +1231,80 @@ In order to delete the cluster, the client must re-supply those fields.
...
+   :header-rows: 1
+
+   * - Status Code
+     - Description
+   * - 200 (OK)
+     - Successful
+   * - 404 (NOT FOUND)
+     - If the cluster requested is not found.
+   * - 409 (CONFLICT)
+     - If the cluster is in the process of performing some other action.
```
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst:1257]: what other actions?  Is it actually possible to return a 409?

File: coopr-docs/docs/source/rest/clusters.rst [hunk 5]
```diff
@@ -1231,3 +1231,80 @@ In order to delete the cluster, the client must re-supply those fields.
...
```
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst:1293]: what other actions, is it actually possible to return a 409?

File: coopr-docs/docs/source/rest/clusters.rst [hunk 6]
```diff
@@ -1231,3 +1231,80 @@ In order to delete the cluster, the client must re-supply those fields.
...
+               "expireTime": 1234567890
+           }'
+        http://<server>:<port>/<version>/clusters/<cluster-id>/pause
+
+Resume a Cluster Job
+=============================
+To resume a cluster job that was paused, make a POST HTTP request to URI:
+::
+
+ /clusters/{cluster-id}/pause
```
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst]: pause -> resume

File: coopr-docs/docs/source/rest/clusters.rst [hunk 7]
```diff
@@ -1231,3 +1231,80 @@ In order to delete the cluster, the client must re-supply those fields.
...
+
```
Comment (albertshau) [inline coopr-docs/docs/source/rest/clusters.rst:1281]: similar to above, it is useful to describe who is allowed to resume.

Admins and cluster owners are allowed to resume a paused cluster job.  If there is paused job currently in progress, the request succeeds and returns a 200.

**Comment by ytolochkevych** [comment]
@albertshau right,  is it actually impossible to return a 409, removed from docs

**Comment by wolf31o2** [comment]
:+1: LGTM... I'll merge

**Pull request merged**